README.md

```diff
@@ -16,6 +16,7 @@ Barista introduces a discoverable API for the Espresso features. So, you and all
 ```java
 // Click widgets
 click(R.id.button);
+click(R.stiring.button_text); // This will click a view that uses `<string name="button_text">Button text</string>`
```

Comment:
There's a typo: `stiring`. In additon, we're introducing Barista to developers. Let's avoid overexplaining things like this... I mean, `click(R.string.something)` is going to click the view with the text `R.string.something` for sure.